src/Search/Indexes/SearchIndex.php

```diff
@@ -501,7 +501,7 @@ protected function _getFieldValue($object, $field)
         $errorHandler = function ($no, $str) {
             throw new Exception('HTML Parse Error: ' . $str);
         };
-        set_error_handler($errorHandler, E_ALL);
+        set_error_handler($errorHandler, E_ALL & ~(E_DEPRECATED | E_USER_DEPRECATED));
```

Comment:
That `$errorHandler` is weak ... we need to make it smarter so we don't just suppress deprecation warning.